src/StaticATokenLM.sol

```diff
@@ -485,7 +486,10 @@ contract StaticATokenLM is
     uint16 referralCode,
     bool fromUnderlying
   ) internal returns (uint256) {
+    require(assets != 0, StaticATokenErrors.INVALID_ZERO_AMOUNT);
```

Comment:
This requirement seems redundant because if `assets = 0` then `shares` which is calculated by `previewDeposit` will immediately be 0 based on rayDivRoundDown:
```
  function rayDivRoundDown(uint256 a, uint256 b)
    internal
    pure
    returns (uint256)
  {
    return (a * RAY) / b;
  }
```

Reply:
agreed that it's unnecessary, so we removed it